example/imageSender/README.md

```diff
@@ -22,4 +30,30 @@
     |--- source
     +--- CMakeLists.txt
 ```
-7. Use CMAKE to trigger the build
\ No newline at end of file
+9. Use CMAKE to trigger the build
+
+![image](https://github.com/ni/adas-replay-hil-simulator-sdk/assets/95634231/0cdd8b24-f1fc-4f5b-948c-98419b2f7739)
+
+10. From the generated ```Build``` folder select the solution file 
+
+![image](https://github.com/ni/adas-replay-hil-simulator-sdk/assets/95634231/67e86902-e7a0-41d6-8b07-b0c1c6175030)
+
+11. Build the project 
+
+12. Download ```ImageSender.zip``` from https://github.com/ni/adas-replay-hil-simulator-sdk/releases/tag/v23.5.0d102
+
+13. Copy ```imageSender.dll``` to the Debug\Release folder , 
```

Comment:
References to `dll` and `zip` are Windows specific. Need to specify based on OS.